signatures/dilithium/dilithium-shake/impl/avx2.yaml

```diff
@@ -1,5 +1,6 @@
-name: AVX2, vectorized SHAKE
+name: AVX2, using AES from OpenSSL
 platform: c
 type: optimized
 hardware features:
   - avx2
+  - aes-ni
```

Comment:
Don't think so

Reply:
probably I accidentally copied impl of aes to shake version